khelo/assets/locales/app_en.arb

```diff
@@ -137,8 +138,42 @@
 
   "@_MY_CRICKET": {
   },
-  "my_game_teams_tab_title": "Teams",
+  "my_cricket_teams_tab_title": "Teams",
   "my_cricket_screen_title": "My Cricket",
+  "my_cricket_tournament_title": "Tournament",
+
+  "@_ADD_TOURNAMENT": {
+  },
+  "add_tournament_screen_title": "Add Tournament",
+  "add_tournament_name": "Tournament Name",
+  "add_tournament_type": "Tournament Type",
+  "add_tournament_start_date": "Start Date",
+  "add_tournament_end_date": "End Date",
+
+
+  "@_TOURNAMENT_TYPE":{
+  },
+  "tournament_type_knockout": "KnockOut",
+  "tournament_type_miniRobin": "Mini Robin",
+  "tournament_type_boxLeague": "Box League",
+  "tournament_type_doubleOut": "Double Out",
+  "tournament_type_superOver": "Super Over",
+  "tournament_type_bestOf": "Best Of",
+  "tournament_type_gully": "Gully",
+  "tournament_type_mixed": "Mixed",
+  "tournament_type_other": "Other",
+
+  "tournament_type_knockout_description": "Teams face off in a single elimination, with the loser immediately knocked out.",
+  "tournament_type_miniRobin_description": "A smaller round-robin format where each team plays once against all others.",
+  "tournament_type_boxLeague_description": "Teams are divided into groups, with top teams advancing to the knockout stage.",
+  "tournament_type_doubleOut_description": "Teams get two chances before being knocked out, with a winners and losers bracket.",
+  "tournament_type_superOver_description": "A knockout format with a super over to decide tied matches.",
+  "tournament_type_bestOf_description": "Teams play a series of matches, and the first to win the majority is the champion.",
+  "tournament_type_gully_description": "Casual street-style cricket with short games and flexible rules.",
+  "tournament_type_mixed_description": "Teams are randomly mixed, creating fun and unpredictable matches",
```

Comment:
Localisation identifier should follow snake case, rename it with:
tournament_type_**box_league**
tournament_type_**double_out**_description